src/js/utils/ui.js

```diff
@@ -66,6 +66,9 @@ export default class UI extends Events {
     }
 
     destroy() {
+        if (!this.el) {
```

Comment:
`this.off` before condition may be safer in case it returns. Really like this change.

Reply:
The issue is in off. So we need this here.